domaintools/_version.py

```diff
@@ -19,4 +19,4 @@
 OTHER DEALINGS IN THE SOFTWARE.
 
 """
-current = "0.6.1"
+current = "0.6.2"
```

Comment:
I merged back in some documentation changes from main which has a bumped version